src/Libraries/RevitNodesUI/Controls/ComboControl.xaml

```diff
@@ -33,7 +33,7 @@
                   ItemsSource="{Binding Model.DropDownNodeModel.Items}"
                   SelectedIndex="{Binding Model.SelectedIndex, Mode=TwoWay}"
                   Style="{StaticResource RefreshComboBox}"
-                  Margin="0,0,0,0"/>
+                  Margin="4,0,4,0"/>
```

Comment:
With the new Style of Select Button, its width is different with ComboBox, so need to adjust.